app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -209,9 +203,9 @@ class AttendeeFragment : Fragment() {
                 // Do nothing
             }
 
-            override fun onItemSelected(p0: AdapterView<*>?, p1: View?, p2: Int, p3: Long) {
-                selectedPaymentOption = paymentOptions[p2]
-                if (selectedPaymentOption == "Stripe")
+            override fun onItemSelected(p0: AdapterView<*>?, p1: View?, position: Int, p3: Long) {
+                selectedPaymentOption = position
+                if (position == paymentOptions.indexOf(getString(R.string.stripe)))
```

Comment:
Use constant please

Reply:
Can I compare directly with the position? if(position = 0 or 1).

Reply:
No, for this, please remove the string resources and use constants

Reply:
Change it to `selectedOption == STRIPE` and don't use string resources in any of its cases

But I'm merging it now, create another issue to handle it